dbcon/mysql/ha_mcs_client_udfs.cpp

```diff
@@ -294,14 +294,16 @@ extern "C"
 
         try
         {
-            if (getenv("SKIP_OAM_INIT"))
+            // Disable legacy-OAM.
+            //if (getenv("SKIP_OAM_INIT"))
             {
```

Comment:
Consider commenting out this set of braces. I don't see a reason to keep them since there aren't any new variables created in this block.